mappings/14w26a#14w26b.tinydiff

```diff
@@ -1,3 +1,3 @@
 tiny	2	0
 c	net/minecraft/unmapped/C_8253973
-	f	Lnet/minecraft/unmapped/C_1241852;	f_0255726		f_0255726
+	f	Lnet/minecraft/unmapped/C_1241852;	f_0255726		block
```

Comment:
This might be a little too generic of a name for a field in `AnimalEntity`. From a quick glance at the code it looks to have something to do with path finding? The name should reflect what it does since it cannot be inferred in any way.

Reply:
aaand that explains why enigma suggestions need to be confirmed ig...